modules/cxf/src/test/resources/gzip-encoding-conf-httpn.xml

```diff
@@ -24,35 +36,38 @@
                     <spring:bean class="org.mule.runtime.module.cxf.feature.ProxyGZIPFeature"/>
             </cxf:features>
         </cxf:proxy-service>
-        <flow-ref name="copy-properties"/>
```

Comment:
This might have been used so that http inbound headers were sent out in the request, should you be doing that here as well?